DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/javascriptIndexes/javascriptMapIndex/metadata.json

```diff
@@ -0,0 +1,161 @@
+        },
+        {
+            "Title": "JavaScript indexes",
+            "Type": "link",
+            "Url": "https://ravendb.net/docs/article-page/latest/java/indexes/javascript-indexes"
+        },
+        {
+            "Title": "JavaScript indexes - Blog",
+            "Type": "link",
+            "Url": "https://ayende.com/blog/182849-A/ravendb-4-1-features-javascript-indexes"
+        }
+    ],
+
+    "Walkthroughs": [
+        {
+            "Title": "Define the Map Index class",
+            "Slug": "step-1",
+            "DescriptionLines": [
+                "* Create the Map Index class - inherit from `AbstractJavaScriptIndexCreationTask`."
+            ],
+            "Assets": [
+                {
+                    "Title": "Static Indexes Overview Demo",
+                    "Type": "demo",
+                    "Url": "/java/static-indexes/static-indexes-overview"
+                },
+                {
+                    "Title": "Map Indexes",
+                    "Type": "link",
+                    "Url": "https://ravendb.net/docs/article-page/latest/java/indexes/map-indexes"
+                },
+                {
+                    "Title": "JavaScript indexes",
+                    "Type": "link",
+                    "Url": "https://ravendb.net/docs/article-page/latest/java/indexes/javascript-indexes"
+                }
+            ]
+        },
+        {
+            "Title": "The Index Entry class",
+            "Slug": "step-2",
+            "DescriptionLines": [
+                "* The IndexEntry class defines the _index-fields per entry_ in the index.<br>",
+                "* These fields will be indexed and can be queried on (step 4).",
+                "",
+                "* The _content_ for each index-field is configured in the index constructor (see next step)."
+            ],
+            "Assets": [
+                {
+                    "Title": "Map Indexes",
+                    "Type": "link",
+                    "Url": "https://ravendb.net/docs/article-page/latest/java/indexes/map-indexes"
+                },
+                {
+                    "Title": "Static Indexes",
+                    "Type": "link",
+                    "Url": "https://ravendb.net/learn/inside-ravendb-book/reader/4.0/10-static-indexes-and-other-advanced-options#static-indexes-and-other-advanced-options"
+                },
+                {
+                    "Title": "Working with Indexes",
+                    "Type": "link",
+                    "Url": "https://ravendb.net/learn/inside-ravendb-book/reader/4.0/12-working-with-indexes#working-with-indexes"
+                }
+            ]
+        },
+        {
+            "Title": "The index constructor",
+            "Slug": "step-3",
+            "DescriptionLines": [
+                "* Define `setMaps()` which is a _HashSet_ that will contain a single JavaScript map string (Map index), or multiple map string (Multi Map index).",
```

Comment:
or multiple map string => or multiple map strings

Reply:
@Drorlev could you address that ?

Reply:
And these 2 as well:

https://github.com/ravendb/demo/pull/113#discussion_r817090710
https://github.com/ravendb/demo/pull/113#discussion_r817109738